### DISCUSSION TITLE
Access to "community" repo

### DISCUSSION THREAD

**Discussion by rkferreira** [Netflix/conductor#2960] [General]
Hi,

I've seen that "contribs" was moved to "community" repo, but I dont see conductor community repo.

Tks

**Comment by apanicker-nflx** [comment]
@rkferreira The community repo is located at - https://github.com/Netflix/conductor-community 
We are working on the announcement and the releases from the next version will officially separate out the modules between the repos but the maven co-ordinates will be similar.

**Reply by rkferreira** [reply]
Thank you

**Reply by apanicker-nflx** [reply]
For more info, please refer - https://github.com/Netflix/conductor/discussions/2961
